extension/jdtls.ext/com.microsoft.gradle.bs.importer/src/com/microsoft/gradle/bs/importer/GradleBuildServerBuildSupport.java

```diff
@@ -676,6 +676,9 @@ private List<IClasspathEntry> getDependencyJars(DependencyModulesResult dependen
                     attributes.add(optionalAttribute);
                 }
                 attributes.add(buildServerAttribute);
+				attributes.add(JavaCore.newClasspathAttribute("groupId", mavenDependencyModule.getOrganization()));
```

Comment:
What will happen if the value part, (mavenDependencyModule.getOrganization()) is `null`?

Reply:
Hm, good question. Do we want to include attributes only if not null and not empty?

Reply:
Yes, it feels safer to only append attributes when they are not blank.